opensrp-chw/src/nacp/assets/json.form/hivst_mobilization_session.json

```diff
@@ -183,7 +183,7 @@
         "openmrs_entity": "concept",
         "openmrs_entity_id": "kit_code",
         "type": "edit_text",
-        "hint": "Kit code",
+        "hint": "Msimbo wa kitepe",
         "relevance": {
           "step1:kit_used_in_demonstration": {
             "type": "string",
```

Comment:
This needs to be in opensrp-chw/src/nacp/assets/json.form-sw/hivst_mobilization_session.json